app/src/main/java/com/mxt/anitrend/base/custom/fragment/FragmentBase.java

```diff
@@ -238,6 +238,10 @@ protected void setViewModel(boolean stateSupported) {
         }
     }
 
+    public void setFilterable(boolean filterable) {
```

Comment:
I added that so that I can make `isFilterable` true when using it just for StaffPageAdapter. It appears that some of these fragments were used all over the place so I couldn't just hardcode it in onCreate()